packages/launcher-component/src/pickers/example-picker.tsx

```diff
@@ -68,20 +68,21 @@ export const ExamplePicker: Picker<ExamplePickerProps, ExamplePickerValue> = {
             missions!.map((mission, i) => {
               const isSelected = props.value.missionId === mission.id;
               const onChange = (runtimeId = props.value.runtimeId, versionId?) => {
+                const runtime = (mission.runtime! as any).find(r => r.id === runtimeId);
                 if (runtimeId) {
-                  const runtime = (mission.runtime! as any).find(r => r.id === runtimeId);
                   if (!runtime || !runtime.versions || runtime.versions.lenght <= 0) {
                     runtimeId = undefined;
                   } else if(!versionId) {
                     versionId = runtime.versions[0].id;
                   }
                 }
+                const version = runtime && runtime.versions.find(v => v.id === versionId);
                 props.onChange({
                   ...props.value,
                   missionId: mission.id,
                   runtimeId,
                   versionId,
-                  downloadOnly: mission.metadata.istio || false
+                  downloadOnly: (version && version.metadata.runsOn.indexOf('local')) !== -1 || false
```

Comment:
@edewit I think you need to keep the istio check and also look for `none` in `runsOn`:
@gastaldi @quintesse more info?

Reply:
@ia3andy I think that @edewit's approach seems correct. The ISTIO check would be really specific for the UI and I just noticed that some examples do not use tthe `istio:true` flag, like in 
https://github.com/fabric8-launcher/launcher-booster-catalog/blob/master/vert.x/community/istio-routing/booster.yaml

Reply:
Of course, in the example above the catalog needs to be updated to contain the flag, but that's another story :wink:

Reply:
@gastaldi didn't you mention a `runsOn: none`?

Reply:
@ia3andy ah yes, you're right. 

In order to avoid complex discussions about this again, I'd suggest to change the `runsOn` in the ISTIO boosters to `none` and test against `none` in here